src/backends/backends.c

```diff
@@ -244,15 +280,15 @@ int RAI_LoadBackend_Torch(RedisModuleCtx *ctx, const char *path) {
 
     RAI_LoadedBackend backend = {0}; // Initialize all the callbacks to NULL.
 
-    int (*init_backend)(int (*)(const char *, void *));
-    init_backend = (int (*)(int (*)(const char *, void *)))(unsigned long)dlsym(
+    int (*init_backend)(int (*)(const char *, void **));
```

Comment:
Because `RAI_ExportFunc` receives a pointer to a function pointer. This holds for `RM_GetApi` as well, but it is exported to modules as `RedisModule_GetApi` which receives `void*` instead of `void**` (don't no why...)